Services/CO.CDP.OrganisationInformation.Persistence/DatabasePersonInviteRepository.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace CO.CDP.OrganisationInformation.Persistence;
+
+public class DatabasePersonInviteRepository(OrganisationInformationContext context) : IPersonInviteRepository
+{
+    public void Dispose()
+    {
+        context.Dispose();
+    }
+
+    public async Task<PersonInvite?> Find(Guid personInviteId)
+    {
+        return await context.PersonInvites.FirstOrDefaultAsync(t => t.Guid == personInviteId);
+    }
+
+    public async Task<IEnumerable<PersonInvite>> FindByOrganisation(Guid organisationId)
+    {
+        var organisation = await context.Organisations
```

Comment:
Can we optimise these queries here, rather those two statements, it can be merged into one by join

`return await context.PersonInvites.Where(pi => pi.Organisation.guid == organisationId).ToArrayAsync();`